core/object/object.cpp

```diff
@@ -920,6 +920,14 @@ TypedArray<Dictionary> Object::_get_property_list_bind() const {
 	return convert_property_list(&lpi);
 }
 
+bool Object::_property_can_revert_bind(const StringName &p_name) const {
+	return property_can_revert(p_name);
+}
+
+Variant Object::_property_get_revert_bind(const StringName &p_name) const {
+	return property_get_revert(p_name);
+}
```

Comment:
Why do we need those `_bind` methods? Seems like you could expose the original methods directly, which already seem `public`.

Reply:
Can I redirect the question to whoever designed this for other methods? 😅 `set`/`get` have the same setup, though maybe I'm missing some obvious reason why it's required there. I can simplify here, I just went for consistency.

Reply:
`set` and `_set_bind` have a different signature. `set_bind` is needed to remove the optional part we don't want to (and can't) expose:

```cpp
	void set(const StringName &p_name, const Variant &p_value, bool *r_valid = nullptr);
	void _set_bind(const StringName &p_set, const Variant &p_value);
```

Same for the others. But for the two you're adding it's not needed.